README.md

```diff
@@ -125,14 +125,26 @@ import { setupBrowserFakes } from 'ember-browser-services/test-support';
 
 module('Scenario Name', function (hooks) {
   setupBrowserFakes(hooks, {
-    window: { location: { href: 'https://crowdstrike.com' } },
+    window: {
+      location: { href: 'https://crowdstrike.com' },
+      fetch: async (url) => ({ url })
```

Comment:
Shouldn't the faked version match the signature of the original function? 

e.g. shouldn't this return something matching:

```ts
interface Response extends Body {
    readonly headers: Headers;
    readonly ok: boolean;
    readonly redirected: boolean;
    readonly status: number;
    readonly statusText: string;
    readonly type: ResponseType;
    readonly url: string;
    clone(): Response;
}

interface Body {
    readonly body: ReadableStream<Uint8Array> | null;
    readonly bodyUsed: boolean;
    arrayBuffer(): Promise<ArrayBuffer>;
    blob(): Promise<Blob>;
    formData(): Promise<FormData>;
    json(): Promise<any>;
    text(): Promise<string>;
}
```

(from `lib.dom`)

Obviously you can't have *the right* body but the `url` on the `Response` would enable the test you've written (and maybe in general the sort of tests that you'd want to fake `fetch` for?)

Reply:
@gnclmorais if you need/want help, lemme know! 

personally, I don't think we should block README updates on something like this, since the underlying implementation is _basically_  YOLO-stubbing